chart/values.yaml

```diff
@@ -0,0 +1,8 @@
+certMonitorController:
+  image: 'registry.local/cert-monitor-controller:latest'
+  imagePullSecret: ''
```

Comment:
as discussed, should not be a string but a struct with registry, name, password and should then be used to create the imgPullSecret